src/main/java/io/strimzi/kafka/quotas/StorageChecker.java

```diff
@@ -77,13 +81,25 @@ public void run() {
         }
     }
 
+    @Deprecated(forRemoval = true, since = "0.3.0")
```

Comment:
TBH, I do not think this is any kind of public API. And even if it is, it is still 0.x` version, so I would say unless it is needed for anything, we should just remove it.